src/components/Uploads/UploadsListItem.js

```diff
@@ -2,14 +2,25 @@ import React from 'react';
 import { useDispatch } from 'react-redux';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import { UploadingProgressBar } from './UploadProgressBar';
 import { UploadsActions } from './UploadsActions';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { colors } from '../../colors';
+import RetryIcon from '@material-ui/icons/CachedRounded';
+import PauseRoundedIcon from '@material-ui/icons/PauseRounded';
 
 const StyledContainer = styled.div`
   display: flex;
   flex-direction: row;
   padding: 0.5em 1em;
   height: 60px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.25px;
```

Comment:
I believe we should have a limited set of font styles defined in the Material UI theme:

https://material-ui.com/customization/typography/#variants

![image](https://user-images.githubusercontent.com/6686679/95429043-7e8bee00-0952-11eb-9f83-2234794d4b6f.png)

And then in the components we would use it with help of [Typography](https://material-ui.com/components/typography/) component like
```
<Typography variant="body2">
  Some text with proper font size, line height, letter spacing
</Typography>
```

@TomaTomoe do we have the "type scale" defined in the Figma design?

https://material.io/design/typography/the-type-system.html#type-scale

![image](https://user-images.githubusercontent.com/6686679/95428843-3d93d980-0952-11eb-80be-4cb9fb716a46.png)

Reply:
Type scale was added to Figma https://www.figma.com/file/FxmwqO6LRg7tOVXrZqMMG0/Neural-Notes-Design-v.0.3?node-id=212%3A836